src/components/FileViewer/FileContextMenu.tsx

```diff
@@ -39,6 +40,7 @@ const FileContextMenu = forwardRef(
     },
     ref: React.Ref<HTMLDivElement>
   ) => {
+    const zipFileExtensions = ["rar", "zip", "7z", "tar", "gz", "xz", "bz2", "tbz2", "tgz", "txz", "tlz", "lz"];
```

Comment:
only rar, zip, gz, tgz are supported

Reply:
I think frontend should still offer the option to unzip, and backend will reject it, and then frontend will catch the error and toast.

It'd be weird if the user right clicks on say `.7z` and the unzip button simply does not appear.